doc/plugins/terminal.rst

```diff
@@ -0,0 +1,51 @@
+###############
+Spyder Terminal
+###############
+
+**Spyder-terminal** is a plugin that allows you to have an integrated Terminal inside Spyder.
```

Comment:
`**Spyder-terminal** is a plugin that allows you to have an integrated system Terminal inside Spyder.`

Reply:
```suggestion
**Spyder-terminal** is a plugin that allows you to have integrated system terminals inside Spyder.
```

I agree with @steff456's suggestion, but with this small improvement because users can have as many terminals as they want.